src/test/java/org/jboss/set/draw/LotteryDrawingTest.java

```diff
@@ -222,7 +252,8 @@ public void testAppendingRepositoryForUnsubcription() throws Exception {
         List<Mail> sent = mailbox.getMailsSentTo(email);
         assertEquals(1, sent.size());
         // Check directly for this URL as we know it is valid one. Using templates might result in a corrupted URL overall
-        assertTrue(sent.get(0).getText().contains("https://github.com/The-Huginn/jira-issue-lottery/blob/main/.github/jira-issue-lottery.yml"));
```

Comment:
this doesn't seem right, as this is a single test with concrete value, with the intention of making sure a valid link is generated